builtin/providers/aws/resource_aws_ami.go

```diff
@@ -506,6 +506,20 @@ func resourceAwsAmiCommonSchema(computed bool) map[string]*schema.Schema {
 			},
 		},
 
+		"encrypted": &schema.Schema{
```

Comment:
I don't feel like these should be added to `resourceAwsAmiCommonSchema`, since they only apply to the `aws_ami_copy` resource. Can we add them where we add `source_ami_region` and `source_ami_id`, instead? 

- https://github.com/hashicorp/terraform/blob/v0.6.16/builtin/providers/aws/resource_aws_ami_copy.go#L16